glass/lensing.py

```diff
@@ -377,10 +385,10 @@ def multi_plane_matrix(
 
 
 def multi_plane_weights(
-    weights: npt.ArrayLike,
-    shells: collections.abc.Sequence[RadialWindow],
+    weights: npt.NDArray[np.float64],
+    shells: list[RadialWindow],
```

Comment:
see above, no need to restrict it to a list (and it's not always a list in practice)